mock/cpp_mock_scenarios/src/collision/crashing_npc.cpp

```diff
@@ -30,10 +29,7 @@ class IdiotNpcScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit IdiotNpcScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "crashing_npc",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.

Reply:
Constructor has map path and osm file parameters again.